services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: MovieSession = None) -> MovieSession:
```

Comment:
Correct your type annotation, you should not expect a `MovieSession` instance as an input for `session_date`. Besides that, you return not an instance of `MovieSession`, but rather a queryset of these instances. Revise your type annotations in the code so the input and return represent expected types.